packages/server/package.json

```diff
@@ -5,21 +5,21 @@
   "main": "index.ts",
   "type": "module",
   "scripts": {
-    "dev": "ts-node index.ts",
+    "dev": "wait-port 8545 && tsx index.ts",
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "license": "ISC",
   "dependencies": {
     "@ethersproject/providers": "^5.7.2",
-    "@latticexyz/common": "2.0.0-alpha.1.149+024f6037",
-    "@latticexyz/dev-tools": "2.0.0-alpha.1.149+024f6037",
-    "@latticexyz/network": "2.0.0-alpha.1.149+024f6037",
-    "@latticexyz/react": "2.0.0-alpha.1.149+024f6037",
-    "@latticexyz/recs": "2.0.0-alpha.1.149+024f6037",
-    "@latticexyz/schema-type": "2.0.0-alpha.1.149+024f6037",
-    "@latticexyz/std-client": "2.0.0-alpha.1.149+024f6037",
-    "@latticexyz/utils": "2.0.0-alpha.1.149+024f6037",
-    "@latticexyz/world": "2.0.0-alpha.1.149+024f6037",
+    "@latticexyz/common": "link:../../../latticexyz/mud/packages/common",
+    "@latticexyz/dev-tools": "link:../../../latticexyz/mud/packages/dev-tools",
+    "@latticexyz/network": "link:../../../latticexyz/mud/packages/network",
+    "@latticexyz/react": "link:../../../latticexyz/mud/packages/react",
+    "@latticexyz/recs": "link:../../../latticexyz/mud/packages/recs",
+    "@latticexyz/schema-type": "link:../../../latticexyz/mud/packages/schema-type",
+    "@latticexyz/std-client": "link:../../../latticexyz/mud/packages/std-client",
+    "@latticexyz/utils": "link:../../../latticexyz/mud/packages/utils",
+    "@latticexyz/world": "link:../../../latticexyz/mud/packages/world",
```

Comment:
run `pnpm mud set-version --link <relative/path/to/mud>` to replace these with the path on your setup